src/data/recipes.tsx

```diff
@@ -1238,6 +1238,141 @@ const recipes: Recipe[] = [
+  },
+  {
+    id: 50,
+    title: "Crash Hot Sweet Potatoes",
+    blurb: `Sweet and spicy potato cakes`,
+    cookTime: "1 hour",
+    source: "http://thecreeksidecook.com/crash-hot-sweet-potatoes/",
+    servings: 4,
+    ingredients: [
+      "1 & ½ tablespoons light brown sugar",
+      "½ teaspoon kosher salt",
+      "½ teaspoon ground cumin",
+      "½ teaspoon New Mexico chili powder",
+      "½ teaspoon smoked sweet paprika",
+      "¼ teaspoon onion powder",
+      "¼ teaspoon chipotle chili powder",
+      "⅛ teaspoon granulated garlic",
+      "10 coarse grinds of black pepper",
+      "2 medium to large sweet potatoes",
+      "2 tablespoons melted butter",
+      "2 tablespoons olive oil"
+    ],
+    steps: `Set a 3 quart sauce pan of water with a teaspoon of kosher salt on a medium flame until it begins to simmer. Oil a large baking sheet, or cover with parchment paper. In a small bowl, mix together all of the dry spices, and set aside. Peel the sweet potatoes, and cut into 1 & ½ inch thick slices. When the water is simmering, add the potato slices and cook for around 10 minutes, until the slices are getting a little soft. Remove the slices to a cooling rack. You can leave them on the rack until they cool off and the outsides are dry, or if you are in a hurry, blot all of the moisture from the surface of the slices. This is important - they have to be dry on the outside or they won't get crispy. Start the oven preheating to 400º Lay the slices of sweet potato on the prepared pan, and use a water glass to gently compress each slice - you want them to flatten out and the edges should get kind of ruffly. Mix the butter and oil together. Using a pastry brush or the back of a spoon, spread a thin layer of the oil and butter mixture on each slice. Sprinkle some of the spice mixture over each slice, and press onto the surface lightly so that it sticks to the potato slices. Gently turn oven each slice, and repeat the oil/butter and dry spice application as you did on the first side. Roast in the oven for about 15 minutes, until the bottom of the slices is getting caramelized and crispy. Carefully flip over each slice and roast another 10 minutes or so, until both sides are crispy and the spice mixture has melted together and turned all nicely brown. Serve immediately.`
+  },
+  {
+    id: 51,
+    title: "Sweet Potato Ravioli",
+    blurb: `A little time consuming but tasty`,
+    cookTime: "1 hour",
+    source:
+      "https://tastykitchen.com/recipes/main-courses/sweet-potato-ravioli-with-pecans-and-herb-brown-butter-sauce/",
+    servings: 4,
+    ingredients: [
+      "FOR THE FILLING:",
+      "1 pound Sweet Potato, Peeled And Cubed",
+      "2 Tablespoons Butter",
+      "½ cups Grated Parmesan Cheese",
+      "½ cups Dried Bread Crumbs",
+      "Salt And Pepper, to taste",
+      "FOR THE PASTA:",
+      "2-½ cups All-purpose Flour",
+      "1 cup Hot Water",
+      "FOR THE SAUCE:",
```

Comment:
Is this intentional?  It's not an ingredient but it's going to show up in the ingredients.  I'd recommend a feature change if we want to support headers in the ingredient list